x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/columns.tsx

```diff
@@ -211,15 +216,24 @@ export const getColumns = ({
     {
       field: 'tags',
       name: i18n.COLUMN_TAGS,
-      render: (value: Rule['tags']) => (
-        <TruncatableText data-test-subj="tags">
-          {value.map((tag, i) => (
-            <EuiBadge color="hollow" key={`${tag}-${i}`}>
-              {tag}
-            </EuiBadge>
-          ))}
-        </TruncatableText>
-      ),
+      render: (value: Rule['tags']) => {
+        if (value != null && value.length > 0) {
+          return (
+            <TagWrapper data-test-subj="tags">
+              {value.map((tag: string, i: number) => (
+                <EuiBadge
+                  color="hollow"
+                  key={`${tag}-${i}`}
+                  data-test-subj={`rules-table-column-tags-${i}`}
```

Comment:
Could we add a quick test in `x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/columns.test.tsx` that tests that all tags are rendered/visible?